Makefile

```diff
@@ -19,3 +19,10 @@ linux_compile:
 .PHONY: generate_idl
 generate_idl:
 	protoc -I ./vendor/github.com/lyft/flyteidl/protos/ -I ./protos/idl/. --go_out=plugins=grpc:protos/gen ./protos/idl/service.proto
+
+.PHONY: generate
+generate:
+	which pflags || (go get github.com/lyft/flytestdlib/cli/pflags)
+	which mockery || (go get github.com/enghabu/mockery/cmd/mockery)
+	which enumer || (go get github.com/alvaroloes/enumer)
```

Comment:
We currently don't use `enumer` in datacatalog, let's we remove this line for now